### DISCUSSION TITLE
Permission to wenbinyugroup/GitBranchingDemo.git denied

### DISCUSSION THREAD

**Discussion by sichenliuPU** [wenbinyugroup/GitBranchingDemo#15] [Q&A]
![image](https://github.com/wenbinyugroup/GitBranchingDemo/assets/160047901/38d9c7fa-8429-47e9-831b-0c12491c357b)
I cannot resolve this issue. @banghuazhao

**Comment by sichenliuPU** [comment]
I have yet to figured out why. Could you help me with that? @banghuazhao

**Reply by banghuazhao** [reply]
Hey @sichenliuPU, could you try to push to your repo? Because this issue is too general, it might be either the permission issue in wenbinyugroup or your setting.
I suggest creating your own private git repo first and then trying to push to it to check if it can succeed.
Here are some solutions FYI: https://stackoverflow.com/questions/47465644/github-remote-permission-denied

**Reply by banghuazhao** [reply]
Hi @unitasium @duhd1993 , do you have some suggestions? Since you have successfully created a pull request, maybe you can share your experience of this issue if possible.

**Reply by unitasium** [reply]
These two pages might be also helpful:
- https://stackoverflow.com/questions/7438313/pushing-to-git-returning-error-code-403-fatal-http-request-failed
- https://docs.github.com/en/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors
